app/client/cli/event.go

```diff
@@ -155,7 +155,7 @@ func (c *Client) CmdEvents() *cobra.Command {
 
 			f := formatter{
 				header: []string{"EVENT TAG", "NAME", "# TEAM", "# EXERCISES", "CAPACITY", "CREATION TIME"},
```

Comment:
We should remove the # before EXERCISES because it's not a count anymore.

Reply:
Yeah you are right